tests/PHPUnit/PredisConnectionTestCase.php

```diff
@@ -511,10 +531,11 @@ public function testThrowsExceptionOnReadWriteTimeout()
     /**
      * @medium
      * @group connected
-     * @expectedException \Predis\Protocol\ProtocolException
      */
     public function testThrowsExceptionOnProtocolDesynchronizationErrors()
     {
+        $this->expectException('Predis\Protocol\ProtocolException');
```

Comment:
Personally, I would always set the exception right before the line that will cause it.
But I can understand if that's too much nowfor this PR.

Reply:
I'd prefer to leave it as is anyway, at least for a while, mostly because it looks kind of similar to the previous approach of having expected exceptions defined on top of the test method by using attributes.